src/GameData.ts

```diff
@@ -0,0 +1,265 @@
+                id: c.id,
+                name: c.name,
+                key: c.alias,
+                skins: [],
+            }));
+
+        const skinDataUrl = "https://raw.communitydragon.org/latest/plugins/rcp-be-lol-game-data/global/default/v1/skins.json";
+        const skins = await (await fetch(skinDataUrl)).json() as SkinData[];
+
+        (champions as ChampionData[]).map(c => c.id)
+            .filter(id => id > 0)
+            .map(i => i.toString())
+            .forEach(id => {
+                const skinKeys = Object.keys(skins)
+                    .filter(k => k.startsWith(id))
+                    .filter(k => (k.length - id.length) === 3)
+                    .map(k => +k);
+
+                skinKeys.forEach(key => {
+                    const data = skins[key];
+                    returnData.filter(r => r.id === +id)[0].skins.push({ ...data });
+                });
+            });
+
+        return returnData;
+    }
+
+    public async loadPerkData(): Promise<PerkData[]> {
+        const perkDataUrl = "https://raw.communitydragon.org/latest/plugins/rcp-be-lol-game-data/global/default/v1/perks.json";
+        return await (await fetch(perkDataUrl)).json() as PerkData[];
+    }
+
+    public async loadItemData(): Promise<ItemData[]> {
+        const returnData: ItemData[] = [];
+
+        const itemDataUrl = "https://raw.communitydragon.org/latest/plugins/rcp-be-lol-game-data/global/default/v1/items.json";
+        const items = await (await fetch(itemDataUrl)).json();
+        items.forEach((c: ItemData) => returnData.push({ ...c }));
+        returnData.forEach((c: ItemData) => { c.to = []; c.from = []; });
+
+        items.forEach((i: any) => {
+            const item = returnData.filter(r => r.id === +i.id)[0];
+
+            i.from.forEach((f: number) => {
+                const otherItem = returnData.filter(r => r.id === f)[0];
+                if (otherItem !== undefined) {
+                    item.from.push(otherItem.name);
+                }
+            });
+
+            i.to.forEach((f: number) => {
+                const otherItem = returnData.filter(r => r.id === f)[0];
+                if (otherItem !== undefined) {
+                    item.to.push(otherItem.name);
+                }
+            });
+        });
+
+        return returnData;
+    }
+
+    public onLookup(message: Discord.Message, isAdmin: boolean, command: string, args: string[]) {
+        if (message.cleanContent.length === 0) {
+            let response = `I have info on the following categories; \`item\`,\`perk\`,\`champion\``;
+            response += `type !${command} {search_type} {search_term} to use it`;
+            message.channel.send(response);
+            return;
+        }
+
+        if (!["item", "perk", "champion"].some(i => i === args[0])) {
+            message.channel.send(`I'm sorry. I'm unable to parse the category \`${args[0]}\` at this moment. If you want it added, contact a guru`);
```

Comment:
"If you think it should be added, please contact a guru." (better grammar, actual period at the end of the sentence)